### ISSUE TITLE
Custom output for -lookup "person"

### ISSUE THREAD

**Issue by tehhowch** [MCOfficer/EndlessSky-Discord-Bot#20]
most persons have phrases associated.

suggest we generate a random phrase, since they often do not have descriptions
could be handled by calling a command `-quote`, which references the hails.txt and persons.txt files on startup to generate the relevant datasets.

e.g. `-lookup captain nate` -> james prints the image it finds (done), then sees that `GetDataType` was `"person"` and then instead of doing `outputhelper(channel, output.substring(output.indexof(\tdescription).....`, he calls `GenerateQuote("captain nate")` and returns that value.
one could shortcut the image printing by just saying `-quote captain nate`

**Comment by MCOfficer** [comment]
fixed by #23?

**Comment by tehhowch** [comment]
No, #23 just added the normal handling, treating persons like ships. first step is a phrase generator class, then after that is new command for `-quote`
I won't be doing this until September

**Comment by MCOfficer** [comment]
np, take your time
